src/features/features/member-count.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (client) => {
+    const channelId = '';
+
+    const updateMembers = async (guild) => {
+      // const mchannel = await getMemberCountChannel(guild.id);
+      const channel = guild.channels.cache.get(channelId);
+      if (channel == undefined) return;
+      channel.setName(`Members: ${guild.memberCount.toLocaleString()}`);
+    };
+
+    client.on('guildMemberAdd', (member) => updateMembers(member.guild));
+    client.on('guildMemberRemove', (member) => updateMembers(member.guild));
```

Comment:
Please move these to their desired file (`./src/events/guildMemberAdd`, `./src/events/guildMemberAdd`)